src/main/resources/org/jenkinsci/plugins/ewm/steps/ExwsAllocateStep/help-selectedRun.html

```diff
@@ -0,0 +1,18 @@
+<div>
+    <p>
+        If you provide this parameter, the <i>exwsAllocate</i> step will allocate the first workspace used by the
```

Comment:
Interesting feature.
When is allocated, each workspace has a unique generated id. So maybe, instead of specifying the Nth allocated workspace, the user can simply specify the id of the workspace to be selected from the upstream job.
Here is the issue to keep track of it: https://issues.jenkins-ci.org/browse/JENKINS-37079.

In another train of thoughts: if I'll integrate the fingerprint engine for each allocated workspace, this can be simplified. The user can simply specify the fingerprint id of the workspace, without the need to specify the upstream job.